### PR TITLE
(SEC-941) Update honeysql to latest 1.x series

### PR DESCRIPTION
**Pull request by donoghuc**
Pull in security fixes in 1.x series.

Please add all notable changes to the "Unreleased" section of the CHANGELOG.

### CODE REVIEW

**Comment by steveax** [comment]
Nits: `SECT` -> `SEC` and add an entry to the CHANGELOG

**Review by steveax — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- update honeysql to 1.0.461 to distribute latest version to all consuming projects
```
Comment (mwaggett) [inline CHANGELOG.md:2]: Can you update this to mention that this is to resolve security issues?

also (nit) should have a newline between this and the next section.

**Review by mwaggett — COMMENTED** [review]
running Froyo's stuff now

**Review by mwaggett — APPROVED** [review]
looks fine for puppetserver#6.x and pe-pse#2019.8.x

**Comment by donoghuc** [comment]
@austb and @jonathannewman this going to work for your teams?

**Review by jonathannewman — APPROVED** [review]

**Review by austb — APPROVED** [review]
This will break PuppetDB, but we need to take it up.

**Pull request merged**